tests/sentry/api/endpoints/test_user_authenticator_details.py

```diff
@@ -351,22 +351,26 @@ def test_require_2fa__can_delete_last_auth_superuser(self):
         superuser = self.create_user(email="a@example.com", is_superuser=True)
         self.login_as(user=superuser, superuser=True)
 
-        # enroll in one auth method
-        interface = TotpInterface()
-        interface.enroll(self.user)
-        assert interface.authenticator is not None
-        auth = interface.authenticator
+        new_options = settings.SENTRY_OPTIONS.copy()
+        new_options["sms.twilio-account"] = "twilio-account"
```

Comment:
The value of this doesn't matter, it just needs to be set (ex: `new_options["sms.twilio-account"] = ""` is fine too). I kept it matching other test cases in this file but open to changing it